src/styles/page.scss

```diff
@@ -0,0 +1,44 @@
+.page {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+
+  .productNameFont {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $boldTextColor;
+    margin-top: 40px;
+    width: $defaultWidth;
+    margin-bottom: 0;
+  }
+
+  .productCode {
+    display: flex;
+    justify-items: flex-start;
+    width: 100%;
+  }
+
+  .productCodeFont {
+    margin: 0;
+    margin-left: 16px;
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $thinTextColor;
```

Comment:
Ensure that the variable '$thinTextColor' is defined in your SCSS files. If it is not defined, the SCSS compilation will fail.